### PR TITLE
fix: many dependent tasks depend on a node, task status bug #2112

### PR DESCRIPTION
**Pull request by wen-hemin**
Please see #2112 for details.

### CODE REVIEW

**Comment by dailidong** [comment]
@lenboo please review this pr

**Comment by dailidong** [comment]
@lenboo

**Comment by gabrywu** [comment]
@Rubik-W Can you resolve the 0.0% Coverage which just config the new UT class to root pom?

**Comment by Rubik-W** [comment]
This PR has no progress due to inconsistent review opinions, 
Detail：https://github.com/apache/incubator-dolphinscheduler/issues/2112

**Comment by qiaozhanwei** [comment]
@lenboo please track this pr , Thx

**Comment by CalvinKirs** [comment]
Hi, please check the reason for the failure of sonar. If there is no update after 7 days, the PR will be closed, and you can reopen it when your problem is solved.

File: dolphinscheduler-service/src/test/java/org/apache/dolphinscheduler/service/process/ProcessServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.apache.dolphinscheduler.dao.entity.TaskInstance;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+
+/**
+ * ProcessService Tester
+ */
+public class ProcessServiceTest {
```
Comment (CalvinKirs) [inline dolphinscheduler-service/src/test/java/org/apache/dolphinscheduler/service/process/ProcessServiceTest.java:29]: Add the test file to the pom file to ensure that the ut will be executed

**Comment by lenboo** [comment]
i think this pr should be closed because of no consensus for now.

**Pull request closed without merging**